.pending/features/sdk/4144-Configurable-Be

```diff
@@ -0,0 +1 @@
+#4144 Allow for configurable BIP32 HD path and coin type.
```

Comment:
```suggestion
#4144 Allow for configurable BIP44 HD path and coin type.
```
Last tiny change, can we change this to BIP44? 
We are not actually supporting generic BIP32 paths or testing that actually works. It is likely that any BIP32 path beyond BIP44 will not work well right now.